### PR TITLE
encrypted export UI changes

### PR DESCRIPTION
**Pull request by cd-bitwarden**
## Type of change

```
- [ x ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Update UI styling to properly use components

## Code changes

Updated the buttons to properly use bit-button on Export.component.html 

## Screenshots

[[](url)](https://files.slack.com/files-pri/T2PL94Q15-F045316SVMM/screen_shot_2022-10-06_at_6.32.40_pm.png)

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: apps/web/src/app/tools/import-export/export.component.html [hunk 1]
```diff
@@ -84,73 +84,48 @@ <h1>{{ "exportVault" | i18n }}</h1>
...
-              />
-
-              <div class="input-group-append">
-                <button
-                  bitSuffix
-                  bitButton
-                  buttonType="secondary"
-                  appStopClick
-                  appA11yTitle="{{ 'toggleVisibility' | i18n }}"
-                  [attr.aria-pressed]="showFilePassword"
```
Comment (differsthecat) [inline apps/web/src/app/tools/import-export/export.component.html:105]: Why were the `appA11yTitle` and `attr.aria-pressed` removed? Does the button in the component library handle those kind of things inhererntly?

File: apps/web/src/app/tools/import-export/export.component.html [hunk 2]
```diff
@@ -84,73 +84,48 @@ <h1>{{ "exportVault" | i18n }}</h1>
...
+            </bit-hint>
+          </bit-form-field>
+
+          <bit-form-field class="tw-w-full">
+            <bit-label>{{ "confirmFilePassword" | i18n }}</bit-label>
+            <input
+              id="encrypted_export_input_confirm_file_password"
+              bitInput
+              type="{{ showConfirmFilePassword ? 'text' : 'password' }}"
+              id="confirmFilePassword"
```
Comment (Thomas-Avery) [inline apps/web/src/app/tools/import-export/export.component.html:114]: Looks like multiple ids are set on this input.

File: apps/web/src/app/tools/import-export/export.component.html [hunk 3]
```diff
@@ -84,73 +84,48 @@ <h1>{{ "exportVault" | i18n }}</h1>
...
+          <bit-form-field class="tw-w-full">
+            <bit-label>{{ "filePassword" | i18n }}</bit-label>
+            <input
+              id="encrypted_export_input_file_password"
+              bitInput
+              type="{{ showFilePassword ? 'text' : 'password' }}"
+              formControlName="filePassword"
+              name="password"
+            />
+            <button type="button" bitSuffix bitButton (click)="toggleFilePassword()">
```
Comment (coltonhurst) [inline apps/web/src/app/tools/import-export/export.component.html:96]: I noticed some html attributes for the button were removed. Do we want these gone?

Examples: `appA11yTitle="{{ 'toggleVisibility' | i18n }}"` or `[attr.aria-pressed]="showFilePassword"`, etc

File: apps/web/src/app/tools/import-export/export.component.html [hunk 4]
```diff
@@ -84,73 +84,48 @@ <h1>{{ "exportVault" | i18n }}</h1>
...
+            <bit-label>{{ "confirmFilePassword" | i18n }}</bit-label>
+            <input
+              id="encrypted_export_input_confirm_file_password"
+              bitInput
+              type="{{ showConfirmFilePassword ? 'text' : 'password' }}"
+              id="confirmFilePassword"
+              formControlName="confirmFilePassword"
+              name="confirmFilePassword"
+            />
+            <button type="button" bitSuffix bitButton (click)="toggleConfirmFilePassword()">
```
Comment (coltonhurst) [inline apps/web/src/app/tools/import-export/export.component.html:118]: Same as above, some html attributes on the button now gone, do need them?

**Review by coltonhurst — COMMENTED** [review]
Left some general comments.

**Update**: I started the review before @Thomas-Avery and @differsthecat had commented, looks like some overlap but feel free to answer either and just reference the overlapped comments when resolving.

File: apps/web/src/app/tools/import-export/export.component.html (hunk 2 above)
Comment (cd-bitwarden) [reply apps/web/src/app/tools/import-export/export.component.html:114]: thanks!

File: apps/web/src/app/tools/import-export/export.component.html (hunk 1 above)
Comment (cd-bitwarden) [reply apps/web/src/app/tools/import-export/export.component.html:105]: I noticed we didnt use it on the login page where we are using this component so I assumed its not needed

Comment (cd-bitwarden) [reply apps/web/src/app/tools/import-export/export.component.html:105]: added it back

File: apps/web/src/app/tools/import-export/export.component.html (hunk 2 above)
Comment (cd-bitwarden) [reply apps/web/src/app/tools/import-export/export.component.html:114]: removed the wrong one

File: apps/web/src/app/tools/import-export/export.component.html (hunk 3 above)
Comment (cd-bitwarden) [reply apps/web/src/app/tools/import-export/export.component.html:97]: I added them back

File: apps/web/src/app/tools/import-export/export.component.html (hunk 4 above)
Comment (cd-bitwarden) [reply apps/web/src/app/tools/import-export/export.component.html:120]: Added them back :)

File: apps/web/src/app/tools/import-export/export.component.html (hunk 3 above)
Comment (coltonhurst) [reply apps/web/src/app/tools/import-export/export.component.html:97]: Great, thanks!

File: apps/web/src/app/tools/import-export/export.component.html (hunk 4 above)
Comment (coltonhurst) [reply apps/web/src/app/tools/import-export/export.component.html:120]: Great, thanks!

File: apps/web/src/app/tools/import-export/export.component.html [hunk 5]
```diff
@@ -84,73 +84,50 @@ <h1>{{ "exportVault" | i18n }}</h1>
...
-            <div class="small text-muted">
+          <bit-form-field class="tw-w-full">
+            <bit-label>{{ "filePassword" | i18n }}</bit-label>
+            <input
+              bitInput
+              type="{{ showFilePassword ? 'text' : 'password' }}"
+              formControlName="filePassword"
+              name="password"
+              appA11yTitle="{{ 'toggleVisibility' | i18n }}"
+              [attr.aria-pressed]="showFilePassword"
```
Comment (differsthecat) [inline apps/web/src/app/tools/import-export/export.component.html:95]: Do we want an `id` on this input, like the one below?

File: apps/web/src/app/tools/import-export/export.component.html (hunk 4 above)
Comment (coltonhurst) [reply apps/web/src/app/tools/import-export/export.component.html:120]: Did you add them back in the wrong place by accident? They are now on the `input` when previously they were on the `button`

File: apps/web/src/app/tools/import-export/export.component.html (hunk 5 above)
Comment (cd-bitwarden) [reply apps/web/src/app/tools/import-export/export.component.html:95]: yes :(

**Review by differsthecat — APPROVED** [review]

File: apps/web/src/app/tools/import-export/export.component.html (hunk 4 above)
Comment (cd-bitwarden) [reply apps/web/src/app/tools/import-export/export.component.html:124]: thank you 🤣

Comment (differsthecat) [reply apps/web/src/app/tools/import-export/export.component.html:124]: It looks like they are on `i` now instead of button!

**Comment by Hinton** [comment]
Are we intending to merge this to `rc` instead of `master`?

**Pull request closed without merging**